admin/Makefile

```diff
@@ -43,7 +43,8 @@ ${BUILD_DIR}/requirements.timestamp: ${BUILD_DIR}/venv.timestamp ../commons/setu
 	${VENV}/bin/pip install -e "."
 	touch $@
 
-${BUILD_DIR}/requirements-dev.timestamp: ${BUILD_DIR}/venv.timestamp setup.py
+${BUILD_DIR}/requirements-dev.timestamp: ${BUILD_DIR}/venv.timestamp requirements.txt setup.py
```

Comment:
requirements.txt should be in ${BUILD_DIR}/requirements.timestamp not in ${BUILD_DIR}/requirements-**dev**.timestamp